src/sections/Header.html

```diff
@@ -0,0 +1,29 @@
+<section
+  class="flex flex-row md:justify-between md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+>
+  <div class="flex flex-col text-[1rem] md:text-[1.6rem] lg:text-[1.5rem]">
+    <p>DE</p>
+    <p>EN</p>
+  </div>
+  <div class="pl-[25%] lg:pl-[20%] xl:pl-[40%] md:mr-24">
+    <h1 class="font-semibold text-[1.5rem] lg:text-[3rem] md:text-[3rem]">
+      Baumeister
+    </h1>
+  </div>
+  <div class="text-[1.5rem] pl-[33%] lg:pl-[25%] lg:flex xl:flex lg:flex-col xl:flex-col hidden">
+    <p class="ml-14">About</p>
+    <p class="ml-8">Services</p>
+    <p>References</p>
+    <p class="ml-9">Contact</p>
+  </div>
+<span class="md:pt-4" >
+      <img
+    src="/Images/Group 4.png" 
+    class="pl-[4%] lg:hidden ml-12 md:ml-12 md:object-cover "
+  />
```

Comment:
The navbar is not responsive on the desktop breakpoint.
Try using flex alignments instead of paddings and margins.